ampersand-view.js

```diff
@@ -297,6 +289,22 @@ assign(View.prototype, {
         this.on('change', action, this);
     },
 
+    // Parser the declarative subview definition.
+    // Overload this method to create your own declarative subview style
+    _parseSubviewOpts: function (subview, name) {
```

Comment:
We probably won't publish if we know there are follow up commits coming for it, so we may as well wait to just get it all in.  Sound ok?